### ISSUE TITLE
Our website down

### ISSUE THREAD

**Issue by bao-qian** [jtreanor/SummerCourses#30]
see http://cs1.ucc.ie/~jct1/git_call/
February 28, 2013, 6:48 pm

``` bash
Running: rake assets:precompile
rake aborted!
Can't connect to MySQL server on '127.0.0.1' (111)
/tmp/build_3ima7faitgdd2/vendor/bundle/ruby/1.9.1/gems/mysql2-0.3.11/lib/mysql2/client.rb:44:in `connect'
/tmp/build_3ima7faitgdd2/vendor/bundle/ruby/1.9.1/gems/mysql2-0.3.11/lib/mysql2/client.rb:44:in `initialize'
/tmp/build_3ima7faitgdd2/vendor/bundle/ruby/1.9.1/gems/activerecord-3.2.12/lib/active_record/connection_adapters/mysql2_adapter.rb:16:in `new'
/tmp/build_3ima7faitgdd2/vendor/bundle/ruby/1.9.1/gems/activerecord-3.2.12/lib/active_record/connection_adapters/mysql2_adapter.rb:16:in `mysql2_connection'
.
.
.
.

```

**Comment by jtreanor** [comment]
Fixed in jtreanor/SummerCourses@2fd3322dabdd011ec6c81b4664add085e4b1276f

Changes made in  jtreanor/SummerCourses@dbf1f3d6ea4ff37e25294123833c83b09d129979 made active admin not compile.

**Comment by bao-qian** [comment]
But only home page works.

**Comment by jtreanor** [comment]
It all works now. I just had to do a db:reset

**Comment by bao-qian** [comment]
The ![log said](http://cs1.ucc.ie/~jct1/git_call/)

```
Can't connect to MySQL server on '127.0.0.1' (111)
/tmp/build_3ima7faitgdd2/vendor/bundle/ruby/1.9.1/gems/mysql2-0.3.11/lib/mysql2/client.rb:44:in `connect'
/tmp/build_3ima7faitgdd2/vendor/bundle/ruby/1.9.1/gems/mysql2-0.3.11/lib/mysql2/client.rb:44:in `initialize'
```

What's the meaning of that?

**Comment by jtreanor** [comment]
I'm not sure. It shouldn't be trying to connect to the MySQL server on localhost and it is in the amazon cloud. It doesn't seem to be causing a problem but we may need to keep an eye on it. If it becomes an issue we may need to switch to postgresql. This would be trivial to do.

**Comment by jtreanor** [comment]
This has been fixed in commit jtreanor/SummerCourses@2fc186feb7028d9d8c9f4afbc4e6f13dcdc08292
